README.md

```diff
@@ -1 +1,187 @@
+
+## 🛠 Built With <a name="built-with"></a>
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby On Rails</summary>
+  <ul>
+    <li><a href="">Postgresql</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add User, Post, Like & Comment**
+- **List all User, Post, Like & Comment**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Not Available]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Mac or PC
+- Install Ruby
+- Understanding OOP principles
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd your-folder
+  https://github.com/ohyajk/my-blog-app.git
+```
+
+### Install
+
+Install this project with:
+
+- GitHub Actions
+- Linters
+  - Rubocop
+- Ruby On Rails
+
+### Usage
+
+- Run the app with `rails c` in the terminal from the root folder of the project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Jitender Kumar**
+
+- GitHub: [@ohyajk](https://github.com/ohyajk)
+- LinkedIn: [@ohyajk](https://www.linkedin.com/in/ohyajk/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add Views in App**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ohyajk/my-blog-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please leave a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for providing the reading materials that aided me during the project development
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **Can I use this code?**
+
+  - yes, it is open source. Feel free to fork it.
+
+- **Can I contribute to this project?**
+
+  - Contact me so I can add you as a collaborator. Alternatively you can leave an issue, it will be well appreciated.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/ohyajk/my-blog-app/blob/Dev/LICENSE) licensed.
```

Comment:
- [ ] it would be better to make sure that the `MIT` link is working and it should take you to the License of the project in order for your project will be professional

Reply:
Fixed